GoogleForCreators/web-stories-wp
# Text Sets: Update rogue text set to use border instead of shapes

## Summary
Updates one of the cover text sets to use border instead of 4 shapes on a word

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
Text set in ticket now uses a border instead of 4 separate shapes

## Testing Instructions
Open editor
-> Add the text set in ticket onto the page
-> see that the `category` element now uses border instead of 4 shapes for it's border

---



Fixes #5840
